infra/src/main/kotlin/fi/fta/geoviite/infra/inframodel/InfraModelConversion.kt

```diff
@@ -296,6 +296,7 @@ fun toGvtGeometryElement(
                 )
             }
         }
+        else -> throw InframodelParsingException("Invalid element")
```

Comment:
Tämä aiheutunee siitä ettei tuo pohjatyyppi ole sealed class... ok, eikä kannata nyt muuttaa. Tuohon poikkeukseen voisi ehkä laittaa jotain tietoa mukaan - tuollaisenaan ei kerro juurikaan mitä meni pieleen jos näin kävisi. Elementin nimi + luokka ```element::class.simpleName``` voisi ohjata oikealle polulle